build/task1-1.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
There's no need to track `build` folder by git. Please make appropriate changes in `.gitignore`.